packages/plugins/src/components/plugin-error-boundary/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export class PluginErrorBoundary extends Component {
+	constructor() {
+		super( ...arguments );
```

Comment:
Why use the deprecated `arguments` instead of passing props?

Reply:
Habits are hard to deprecate 😅

Reply:
Resolved in [a579a80](https://github.com/WordPress/gutenberg/pull/38397/commits/a579a8048cab156190e336bd41fa5e7c20400546)